modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc

```diff
@@ -11,6 +11,32 @@ Start a local container cluster.
 rpk container start [flags]
 ----
 
+== Examples
+
+If you're following the guide xref:get-started:quick-start.adoc[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container start -n 3` to start your containers:
```

Comment:
```suggestion
If you're following the guide xref:get-started:quick-start.adoc#deploy-redpanda[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container start -n 3` to start your containers:
```

Reply:
Checked with Jake how to do it. We need to put #id-of-tab. So in this case it's `xref:get-started:quick-start.adoc#tabs-1-three-brokers`